contributing/content-style-guide.md

```diff
@@ -340,6 +341,75 @@ A table following an H3 heading:
 > | `-readable` | Print output in human readable format. | <ul><li>`true`</li><li>`false`</li></ul> |
 > | `-file` | Print output to file instead of stdout. | Name of the file. |
 
+## Cards
+
+Use the `<Card` component to display content and actions on a single topic. Users should cards easy to scan for relevant and actionable information. Elements, like text, icons, and links can be used inside a card. 
+  
+You can configure a card in 2, 3, 4, or 5-column grids. To maintain a good user experience, 4 or 5-column grids should only be used:
```

Comment:
```suggestion
You can configure a card in 2, 3, 4, or 5-column grids. To maintain a good user experience, be mindful of how the cards are displayed. Cards with multiple paragraphs may not work because the text and cards will appear too squished, making them hard to read and a bad user experience. 

4 or 5-column grids should only be used:
```